addons/sale_management/report/sale_report_templates.xml

```diff
@@ -2,7 +2,13 @@
 <odoo>
 <template id="report_saleorder_document_inherit_sale_management" inherit_id="sale.report_saleorder_document">
     <xpath expr="//div[@name='signature']" position="after">
-        <div t-if="not (doc.sale_order_option_ids and doc.state in ['draft', 'sent'])" class="oe_structure">&#8203;</div>
+        <div
+            t-if="not (doc.sale_order_option_ids and doc.state in ['draft', 'sent'])
+                  or (len(doc.sale_order_option_ids)==1 and doc.sale_order_option_ids[0].is_present)"
```

Comment:
It also should not display this table is all of these options are present in orderline

Also no need to check [0] if you are checking len=1 before